Berksfile

```diff
@@ -1,3 +1,8 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+group :solo do
```

Comment:
I wonder if the idea is similar to managing environments in a gemfile as per this rails example?
https://www.railstutorial.org/book/beginning#sec-heroku_setup